### DISCUSSION TITLE
Output From Configuration Manager

### DISCUSSION THREAD

**Discussion by womeara6** [GerryDazoo/Slinger#461] [General]
Hi Gerry.  I have a bricked sole on my network with parental controls blocking internet access.  I started the Slinger server and attempted to make some connections via VLC.  The server spent the last few days logging activity, and it created a number of "Hacker Alerts" from IP addresses outside of my network.  Is this something I need to be worried about?  

Thanks, in advance.
Bill

**Comment by GerryDazoo** [comment]
You should expect some of these if you have a port forwarding rule on your router to allow unrestricted access from the Internet to your server.

**Reply by womeara6** [reply]
HI Gerry.  I have 3 port forwarding rules, all related to the slingboxes.  Both boxes are restricted to use the internet using parental controls.  the Solo is forwarded to port 5001 and the 500 to port 5201.  I have a rule forwarding port 5678 to a non-existent ip address.

Here is one of the alerts.
Hacker Alert. Invalid Request from  ('43.128.227.146', 42796)
GET / HTTP/1.1
Host: [my external ip]:8080
User-Agent: 'Mozilla/5.0
Accept: */*
Accept-Encoding: gzip

Is this what I should expect?

Thanks,
